### DISCUSSION TITLE
The administrator account has no management permission!

### DISCUSSION THREAD

**Discussion by zshuaia** [requarks/wiki#5605] [Help / Questions] [answered]
Can help me？

just now,
1. I went to the admin screen and changed a URN parameter.
2. I modified the menu permissions for a non-administrator user group.

![image](https://user-images.githubusercontent.com/38149007/188359420-266c40bd-fa6f-4c70-aa5b-a812ad96ccd3.png)
![image](https://user-images.githubusercontent.com/38149007/188359438-b4cba1d3-1c9f-4aba-9e64-1af398655b81.png)

**Comment by NGPixel** [comment]
Logout and login again.

**Reply by zshuaia** [reply]
Logging out and logging back in doesn't work.

**Reply by zshuaia** [reply]
Until I went to the server and restarted the wiki service, re-opened the browser, and it worked fine.

**Comment by zshuaia** [comment, marked as answer]
restarted the wiki service, and it worked fine.
